pending_changelog/1699.txt

```diff
@@ -0,0 +1,2 @@
+`broadcast_node_announcement` has been moved to `PeerManager` from `ChannelManager`.
```

Comment:
nit: Maybe we should use the commit hash instead of PR number, to keep the commit history independent from GH?